tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmData.cs

```diff
@@ -86,7 +86,7 @@ public async Task TestBlockedRequestIpWithOneClickActivation(string test, string
             spans.AddRange(spanAfterAsmDeactivated);
             spans.AddRange(spanAfterAsmDataReactivated);
 
-            await VerifySpans(spans.ToImmutableList(), settings, true);
+            await VerifySpans(spans.ToImmutableList(), settings);
```

Comment:
If I understand correctly, the "init" testing is testing the waf is definitely loaded, by including the `AppSecWafVersion` etc tags right? 

I'm not sure why this should be flaky? 🤔 The tags should always be added, as long as appsec is enabled, it's just a question of whether we scrub them, right? For test classes where appsec is enabled, I don't understand why these tags _wouldn't_ be added?

Reply:
You're correct that the "init" testing checks the initial WAF information like `AppSecWafVersion`, however this info is only added once (in method `SecurityCoordinator.Reporter.ReportWafInitInfoOnce`), so only the first application span receives it while no other spans do. So checking for this info now may be flaky since we're grouping multiple tests into one XUnit class and we can't guarantee which test would run first

Reply:
I guess actually they're going to get added in the alive check anyway, so it's always going to get lost in the "normal" tests 🤔